notebooks/data-sources/gcsweb-ci/prow_archive_discovery.ipynb

```diff
@@ -0,0 +1,1159 @@
+{
```

Comment:
<p>Could the <code>save_to_disk</code>  function in metric template be used here instead of repeating the function content?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/199/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
<p>Possibly it could be reused, but there are a couple log and events data specific changes that would need to be made to the function for it to work here. Plus it would require adding the <code>sys.path.append</code>  block of code at the top of the notebook, which is just as long and doesn't agree well with pre-commit. So its a bit of a trade off either way. </p><p> </p><p>Good question though, and maybe we should start to think about the benefit of a <code>utils.py</code>   but for now I think this works. </p>